>From edca292247fd5fddf8c1f86dd18203c33a71d2c7 Mon Sep 17 00:00:00 2001
From: Exceed <exceed.cedric@gmail.com>
Date: Wed, 27 May 2009 22:01:38 +0000
Subject: [PATCH] 	amaya-11.1-1-i686

        * new package
        * closes FS#3619
---
 source/xapps-extra/amaya/FrugalBuild |   40 ++++++++++++++++++++++++++++++++++
 1 files changed, 40 insertions(+), 0 deletions(-)
 create mode 100644 source/xapps-extra/amaya/FrugalBuild

diff --git a/source/xapps-extra/amaya/FrugalBuild b/source/xapps-extra/amaya/FrugalBuild
new file mode 100644
index 0000000..df718a5
--- /dev/null
+++ b/source/xapps-extra/amaya/FrugalBuild
@@ -0,0 +1,40 @@
+# Compiling Time: 3.59 SBU
+# Maintainer: Cedrick Hannier alias Cedynamix <cedynamix@gmail.com>
+# Contributor: exceed <cedric.archambeau@gmail.com>
+
+pkgname=amaya
+_F_archive_name="amaya-sources"
+pkgver=11.1
+pkgrel=1
+pkgdesc="W3C's web Editor/Browser"
+url="http://www.w3.org/Amaya"
+depends=('gtk+2' 'wxgtk' 'raptor')
+options=('scriptlet')
+groups=('xapps-extra')
+archs=('i686')
+up2date="Flasttar $url/Distribution/"
+source=($url/Distribution/${_F_archive_name}-$pkgver.tgz)
+_F_cd_path="Amaya11.1/Amaya/fwlinux"
+unset MAKEFLAGS
+sha1sums=('7023050223304dc8cf34647639c25a475e3b97f9')
+
+_F_desktop_name="Amaya"
+_F_desktop_icon="/usr/share/$pkgname/Amaya/resources/icons/misc/logo.png"
+_F_desktop_categories="Development;WebDevelopment;"
+
+build()
+{
+	mkdir ${_F_cd_path}
+	Fcd ${_F_cd_path}
+	Fmkdir /usr/share/${pkgname}
+	../configure --prefix=/usr/share/${pkgname} \
+		--enable-system-raptor\
+		--with-gl
+	make || Fdie
+	make prefix=$Fdestdir/usr/share/${pkgname} install || Fdie
+	Fln /usr/share/${pkgname}/Amaya/wx/bin/${pkgname} /usr/bin/${pkgname}
+	Frm /usr/share/${pkgname}/bin
+	Fdesktop2
+}
+
+# optimization OK
-- 
1.6.3.1

